### DISCUSSION TITLE
ApexDoc @description is not detected as ApexDoc command.

### DISCUSSION THREAD

**Discussion by PawelWozniak** [forcedotcom/salesforcedx-vscode#5805] [Ideas]
### Summary

Since PMD require to have @description tag added it seems that is part of ApexDoc standard however it is not detected and colored. 
<img width="113" alt="2021-09-10_10h27_27" src="https://user-images.githubusercontent.com/5740564/132824734-41bdfa7d-7f27-4f39-862d-fc3f61eb86cf.png">

Where in the repository I can find a definition of what is treated as ApexDoc? 
I would fix it by myself.

### Steps To Reproduce:

1. Type:
```
	/**
	 *  @description My description
	 *  @param newRecords trigger.new
	 *  @param oldRecordsMap trigger.oldMap
	 */
```

### Expected result

@description is highlited

### Actual result

@description is not highlited

**Comment by tcampb** [comment]
@PawelWozniak - it looks like this extension uses JavaDoc, which does not support the `@description` tag, instead of ApexDoc:

https://github.com/forcedotcom/salesforcedx-vscode/blob/99d36322dd5679d2cac4ffe929e3f032182b580e/packages/salesforcedx-vscode-apex/src/index.ts#L120-L121

**Comment by PawelWozniak** [comment]
I have reviewed this code but don't have enough knowledge how to extend https://github.com/forcedotcom/salesforcedx-vscode/blob/99d36322dd5679d2cac4ffe929e3f032182b580e/packages/salesforcedx-vscode-apex/src/languageClientUtils/javaDocSymbols.ts#L7

**Comment by smaddox-sf** [comment]
Hi @PawelWozniak - Thanks for reaching out and for the willingness to submit a PR!  Unfortunately, the Apex Language Server where this change would need to be made isn't open source yet.  This is an area we'd love to get out in the open but can't do that until some pieces of the Apex compiler are also open sourced.  I tagged this as feedback so we can consider it for the roadmap.  I will let you know as we get this to open source as well.

**Comment by PawelWozniak** [comment]
@smaddox-sf In the meantime maybe someone from SF could add this?
 It should be a simple change like extending the list of strings that are recognized as ApexDoc tags.
